dataprofiler/tests/profilers/profiler_options/test_categorical_options.py

```diff
@@ -11,15 +11,66 @@ class TestCategoricalOptions(TestBaseInspectorOptions):
     def test_init(self):
         option = self.get_options()
```

Comment:
Another PR, but this is naming I should have caught before:
`get_options` -> `set_and_retrieve_option_values` or something. Needs to be an issue.

Reply:
`get_options` is confusing when we set a value in it.

Reply:
+1